packages/ticker/src/Ticker.js

```diff
@@ -137,6 +137,14 @@ export default class Ticker
          */
         this._protected = false;
 
+        /**
+         * The last time keyframe was executed.
+         * @member {number}
+         * @default -1
+         * @private
+         */
+        this._lastKey = -1;
```

Comment:
Couple of minor comments: Why a default value of -1? Also, could you indicate units in description... milliseconds I assume? And finally maybe “lastFrame” is a better name than “lastKey”

Reply:
👌